docs/modules/ROOT/pages/how-to-guides/testing_applications/enable_snyk_check_for_a_product.adoc

```diff
@@ -0,0 +1,63 @@
+= Enabling a Snyk task
+
+[abstract]
+The `sast-snyk-check` task uses the Snyk Code tool to perform Static Application Security Testing (SAST). 
+Specifically, the Snyk check scans an application's source code for potential security vulnerabilities, 
+including SQL injection, cross-site scripting (XSS), and code injection attack vulnerabilities.
+
+> NOTE: You can run a Snyk task only if you have a Snyk token stored in a namespace secret. 
+You should also include the name of your secret in the *snyk-secret* pipeline parameter.
+
+== Enabling a Snyk task
+
+. Register for a Snyk account or log in at https://app.snyk.io/.
+. Get a Snyk token.
+.. In the lower left of the home page, click your name, then select *Account settings*.
+.. From the Account Settings page, select *General*, which is the default, then *Auth Token*.
+.. Under the *Auth Token* section, click *Click to View* to see the *KEY* value of the automatically generated token.
+. Enable Snyk Code.
+.. From the left panel, go to *Settings* > *Snyk Code*, then scroll to the *Enable Snyk Code* section.
+.. Toggle *Disabled* to *Enabled*.
+.. Click *Save* changes.
+. Add your new secret to your workspace.
+.. Log in to the link:https://console.redhat.com/preview/application-pipeline[Red Hat Hybrid Cloud Console {ProductName} Overview page].
+.. From the left menu, click *Secrets*.
+.. Click *Add secret*.
+.. The *Add secret* page displays options for your new secret. Specify the following:
+... For *Secret for*, select *Build*.
+... From the *Secret type* drop-down menu, choose *Key/value secret*.
+... From the *Secret name* drop-down menu, select *snyk-secret*.
+... Paste your Snyk token into the *Upload the file with value for your key or paste its contents* field.
+... Click *Add secret* to save it.
+. Update your `.tekton` YAML files to specify your Snyk secret.
```

Comment:
line 32: can you just make sure that the spacing around `.tekton` is right? looks..._spacious_, but could just be bc of the backticks.